utils/docker/build-wasinn-ubuntu-openvino.sh

```diff
@@ -2,6 +2,16 @@
 # SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
 # SPDX-FileCopyrightText: 2019-2022 Second State INC
 
+if [[ ! -v "${OPENVINO_VERSION}" ]]; then
```

Comment:
I am not sure if we do want the wasinn/openvino related scripts in the `docker` folder.
Maybe putting these in the script folder is better?

Reply:
The `build.sh` and `build-manylinux.sh` are also in the `docker` folder.
Maybe we can move them into `utils/build-scripts` together in the other PR?

Reply:
Please notice that the `build.sh` and `build-manylinux.sh` are highly related to their corresponding dockerfiles, that's why we put them here. However, the OpenVINO/WASI-NN are not related to these dockerfiles.

Reply:
Done. Moved into `utils/wasi-nn`.